build_scripts/desktop/package.sh

```diff
@@ -357,23 +357,34 @@ for product in ${product_list[*]}; do
       if [[ ${run_in_parallel} -eq 0 ]]; then
         # Run immediately if not set to run in parallel.
         "${merge_libraries_tmp}/merge_${product}.sh"
+      else
+        echo "echo \"${libfile_out}\" DONE" >> "${merge_libraries_tmp}/merge_${product}.sh"
       fi
 done
 
 if [[ ${run_in_parallel} -ne 0 ]]; then
+  # Analytics is the smallest SDK, so it should be the shortest job.
+  shortest=analytics
   echo "There are ${#product_list[@]} jobs to run."
-  echo "Running first job to populate cache, then remaining jobs in parallel..."
-  "${merge_libraries_tmp}/merge_${product_list[0]}.sh"
+  echo "Running shortest job to populate cache, then remaining jobs in parallel..."
+  "${merge_libraries_tmp}/merge_${shortest}.sh"
+  # Zero out the job that we already did.
+  echo "#!/bin/bash" > "${merge_libraries_tmp}/merge_${shortest}.sh"
```

Comment:
Come to think of it I could also just rm the script, lmk if you have a preference.

Reply:
I think it is fine to leave it as it is.